wildnlp/datasets/conll.py

```diff
@@ -8,6 +8,7 @@ class CoNLL(Dataset):
     entity recognition. For details see:
     https://www.clips.uantwerpen.be/conll2003/ner/
     """
+    datasetFileHeader: str
```

Comment:
Could you move this variable explicitly to the constructor and make it the private member?

```suggestion
    def __init__(self):
        self._dataset_file_header: str
        super().__init__()
```